### PR TITLE
added report

### PR DESCRIPTION
**Pull request by bedeho**

### CODE REVIEW

File: reports/archive/2.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+3rd May 2019, 11:30am GMT+1
+
+## Topics
+
+Bedeho presents and approach for how to do rutnime specification up front with the goal of making internal coordination clearer, more error free and ultimately faster by reducing amount ambiguity and mistakes.
+
+## Proposal
+
+https://github.com/bedeho/joystream/blob/rome-spec/testnets/rome/members-module.md
```
Comment (bwhm) [inline reports/archive/2.md]: Fix link.
(It should not point to your branch, and the testnet is/will be renamed with #33.

File: reports/archive/2.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Conclusions
+
+### Bedeho post-mortem opinion
+
+No one was terribly excited at the prospect of doing this sort of specification up front. The general feeling was that it was too detailed, and would take so much time to arrive upon up front. Some wanted a higher level description instead, while other suggested simply diving straight into implementation, but instead doing it in stages where the skeleton of a runtime is reviewed by someone else before full implementations are made. While I can appreciate the concern, I am still worried that we will end up shipping less secure/error free code and that our internal shared understanding of what is being built is going to be unreliable - slowing us down. None the less, I am happy to try this higher level approach on the runtime for now, as I am effectively a lone voice on this - and there are so many unknowns. I am not sure where this preference for more informality leaves us on for other nodes/protocols, but we can revisit this. I think you should also give your input on this when you have time.
+
+
+### Note
+
+If any participant wishes to add their own post-mortem, please open a PR.
```
Comment (bwhm) [inline reports/archive/2.md]: Maybe ask the others to have a look as well instead of this?

File: reports/archive/2.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- Martin
+- Bedeho
+
+## Time and place
+
+3rd May 2019, 11:30am GMT+1
+
+## Topics
+
+Bedeho presents and approach for how to do rutnime specification up front with the goal of making internal coordination clearer, more error free and ultimately faster by reducing amount ambiguity and mistakes.
```
Comment (bwhm) [inline reports/archive/2.md]: *runtime

File: reports/archive/2.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Proposal
+
+https://github.com/bedeho/joystream/blob/rome-spec/testnets/rome/members-module.md
+
+## Conclusions
+
+### Bedeho post-mortem opinion
+
+No one was terribly excited at the prospect of doing this sort of specification up front. The general feeling was that it was too detailed, and would take so much time to arrive upon up front. Some wanted a higher level description instead, while other suggested simply diving straight into implementation, but instead doing it in stages where the skeleton of a runtime is reviewed by someone else before full implementations are made. While I can appreciate the concern, I am still worried that we will end up shipping less secure/error free code and that our internal shared understanding of what is being built is going to be unreliable - slowing us down. None the less, I am happy to try this higher level approach on the runtime for now, as I am effectively a lone voice on this - and there are so many unknowns. I am not sure where this preference for more informality leaves us on for other nodes/protocols, but we can revisit this. I think you should also give your input on this when you have time.
```
Comment (bwhm) [inline reports/archive/2.md]: > I think you should also give your input on this when you have time.

Define "you".

File: reports/archive/2.md (hunk 1 above)
Comment (bedeho) [reply reports/archive/2.md]: Done.

File: reports/archive/2.md (hunk 3 above)
Comment (bedeho) [reply reports/archive/2.md]: Done.

File: reports/archive/2.md (hunk 4 above)
Comment (bedeho) [reply reports/archive/2.md]: Done.

**Pull request merged**